reference/tools/env/virtualbuildenv.rst

```diff
@@ -49,20 +49,24 @@ Generated files
 This generator (for example the invocation of ``conan install --tool-require=cmake/3.20.0@ -g VirtualBuildEnv``)
 will create the following files:
 
-- conanbuildenv-release-x86_64.(bat|sh): This file contains the actual definition of environment variables
+- conanbuildenv-release-x86_64.(bat|ps1|sh): This file contains the actual definition of environment variables
   like PATH, LD_LIBRARY_PATH, etc, and any other variable defined in the dependencies ``buildenv_info``
   corresponding to the ``build`` context, and to the current installed
   configuration. If a repeated call is done with other settings, a different file will be created.
   After the execution or sourcing of this file, a new deactivation script will be generated, capturing the current
   environment, so the environment can be restored when desired. The file will be named also following the
   current active configuration, like ``deactivate_conanbuildenv-release-x86_64.bat``.
-- conanbuild.(bat|sh): Accumulates the calls to one or more other scripts, in case there are multiple tools
+- conanbuild.(bat|ps1|sh): Accumulates the calls to one or more other scripts, in case there are multiple tools
   in the generate process that create files, to give one single convenient file for all. This only calls
   the latest specific configuration one, that is, if ``conan install`` is called first for Release build type,
-  and then for Debug, ``conanbuild.(bat|sh)`` script will call the Debug one.
-- deactivate_conanbuild.(bat|sh): Accumulates the deactivation calls defined in the above ``conanbuild.(bat|sh)``.
+  and then for Debug, ``conanbuild.(bat|ps1|sh)`` script will call the Debug one.
+- deactivate_conanbuild.(bat|ps1|sh): Accumulates the deactivation calls defined in the above ``conanbuild.(bat|ps1|sh)``.
   This file should only be called after the accumulated activate has been called first.
 
+.. note::
+
+    To create ``.ps1`` files it is necessary to use the conf: ``tools.env.virtualenv:Powershell``.
```

Comment:
```suggestion
    To create ``.ps1`` files it is necessary to use the conf: ``tools.env.virtualenv:powershell``.
```

Casing is important, it will fail in uppercase